.github/workflows/governance-checks.yaml

```diff
@@ -35,16 +35,25 @@ jobs:
           path: |
             cache
             reports
-          key: ${{ runner.os }}-governance-cache-${{ github.run_number }}
```

Comment:
@marcomariscal your fix is valid. run_number increments every time -> so a new cache is created